packages/office-ui-fabric-react/src/components/Button/ButtonCore/ButtonCoreVars.scss

```diff
@@ -1,3 +1,16 @@
 $button-core-default-height: 32px;
 $button-core-default-minWidth: 80px;
 $button-core-default-padding: 0 16px;
+
+@mixin core-button-disabled {
```

Comment:
@mikewheaton Only if included right? Given this is a component level file, I suspect this is fine. If this was on the other hand part of Fabric-Core, I definitely agree we should rename.

Please let me know if you strongly disagree, renaming something isn't a big deal after all :)